vmm/src/default_syscalls/x86_64.rs

```diff
@@ -126,6 +126,10 @@ pub fn default_context() -> Result<SeccompFilterContext, Error> {
                 libc::SYS_accept,
                 (0, vec![SeccompRule::new(vec![], SeccompAction::Allow)]),
             ),
+            (
+                libc::SYS_brk,
```

Comment:
You need to add it to the `ALLOWED_SYSCALLS` array too. That array contains the whitelisted syscalls for `seccomp_level=1`.

Reply:
Shouldn't we have a test with seccomp-level=1 as well? In which case this PR would've had failed tests.

Reply:
It would only be really really relevant if we make every functional test run with `seccomp-level=1` also. I think a nicer idea would be to, at some point, generate the syscall list for `seccomp-level=1` from that huge thing we pass to `seccomp-level=2`, so we only need to add/remove syscalls from one place.

Reply:
#920 for generating the simple whitelist.

Reply:
https://github.com/firecracker-microvm/firecracker/issues/921 for adding integration tests with various seccomp configurations.

Reply:
Done.